resources/assets/less/bem/mod.less

```diff
@@ -17,6 +17,18 @@
  */
 
 .mod {
+  .mods & {
```

Comment:
`position: absolute` by itself seems to fix it; I'm not sure on the need for `left`, `top` or `content`